src/services/User.tsx

```diff
@@ -55,7 +55,7 @@ interface GetUserNFTs {
 // get user inMarket offMarket
 export const getUserNFTs = async ({ chainId, owner }: GetUserNFTs) => {
   const userNFTs = await getNFTs({ chainId, owner })
```

Comment:
why we filter with chainid ? is it should be load all inventory item or specific current chosen network.
* iam missing business model in this point